README.md

```diff
@@ -84,88 +84,84 @@ dependencies {
+MNNKit SDK contains the deep learning model of related algorithms and runs on top of MNN engine. As a result it's not traditional image processing and contains all the characteristics of deep learning. During training stage algorithm uses face images in top-down direction as such this model only recognize faces in that same direction. If the input contains faces in a different direction, it might not get detected. We call this ``Content Orientation Sensitive``. In MNNKit, FaceDetection,HandGestureDetection and PortraitSegmentation are all ``Content Orientation Sensitive``.
 
-1. 在MNN引擎执行推理之前，需要对原始的输入做预处理，保证输入数据中的人脸为正向
-2. 推理完成后，产生了基于输入图像（预处理之后的）坐标系的关键点结果
-3. 为了上层渲染方便，SDK会把关键点坐标变换到和屏幕渲染坐标系相同的方向
+1. Prior to inferencing with MNN, pre-processing is needed on the raw input to make sure that faces in the raw input image has the right direction.
+2. The inference process yields results based on the coordinate system of the pre-processed image.
+3. To provide easy-to-use results for the caller, the SDK transforms the coordinate system to the same as the screen rendering coordinate system.
 
->  SDK参数中的inAngle参数就是用来控制将原始输入旋转到"内容正向"的角度，outAngle就是用来变换关键点坐标到屏幕渲染坐标系方向的角度！
+> ``inAngle`` in the SDK arguments is to control the angle to rotate so that the raw input in rotated to the required direction and ``outAngle`` is the angle used to rotate the raw inferenced coordinates to the screen rendering coordinate system.
 
-从上图中可以看出，iOS在后置正向的情况下的输入角度为0，输出角度为0；而安卓输入角度为90，输出角度为0。
+As we can see in the image, for a device in positive direction, the back camera has ``inAngle`` and ``outAngle`` both set to ``0`` for iOS, and for Android it's ``90`` and ``0`` respectively.
 
-#### 3. 结果渲染
+#### 3. Result rendering
 
-SDK检测的结果是图像的特征描述，最简单的描述就是关键点的坐标，这些坐标都是在图像坐标系下产生的，也就是在最终输入到引擎的图片左上角为原点的坐标系下。比如输入的图像是1280*720，那么检测结果也是在这个坐标系下产生。
+The final result of the SDK is the description of the feature points, or the coordinate of the feature points to be precise. All these coordinates are related to a certain coordinate system, a system that uses the left-top point of image as the origin. For example if the input image is of resolution ``1280*720``, then the result coordinates are also based on this exact coordinate system.
 
-工程应用中，最后的结果关键点要显示在用户屏幕上，前端会使用一个用来渲染的"画布"，它可以是一个UI上用来显示的视图或其他组件，画布的尺寸由应用根据视觉设计自己定义。画布的坐标系我们统称为渲染坐标系，这一步要做的就是将关键点坐标从头像坐标系转换到渲染坐标系。
+In real world applications, the resulting feature points are supposed to be displayed on the user's screen. The frontend would uses a canvas for rendering which could be a view in the UI that is used for displaying or it could be some other components. The size of the canvas is specified by the application itself based on UI design. The coordinate system of this canvas is called rendering canvas. What this step is doing is to convert the raw inferenced points to rendering coordinate system.
```

Comment:
```suggestion
In real world applications, the resulting feature points are supposed to be displayed on the user's screen. The frontend would use a canvas for rendering which could be a view in the UI that is used for displaying or it could be some other components. The size of the canvas is specified by the application itself based on UI design. The coordinate system of this canvas is called rendering coordinates. What this step is doing is to convert the feature points from the image's coordinate system to rendering coordinate system.
```